docs/weekly-questions/joes-water.md

```diff
@@ -242,4 +242,33 @@ print("YES" if after_discount_applied > original_profit else "NO")
 
 </p>
 </details>
--->
+
+
+
+## Model Solution 2
+
+<details><summary>Click to reveal</summary>
+<p>
+
+Here's a more cut-down-to-the-minimum solution that may be more readable for some of you:
+
+```py
+joesWater = input().split(' ')
```

Comment:
```suggestion
joes_prices = input().split(' ')
```
`prices` is more accurate. In accordance with [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), variable names should be snake_cased.

Reply:
Why was this marked as resolved without comment? (Ditto with the others.)

Reply:
Was about to make changes.